integrations/object_store/src/lib.rs

```diff
@@ -31,16 +31,11 @@
 //!
 //! #[tokio::main]
 //! async fn main() {
-//!     let builder = S3::from_map(
-//!         vec![
-//!             ("access_key".to_string(), "my_access_key".to_string()),
-//!             ("secret_key".to_string(), "my_secret_key".to_string()),
-//!             ("endpoint".to_string(), "my_endpoint".to_string()),
-//!             ("region".to_string(), "my_region".to_string()),
-//!         ]
-//!         .into_iter()
-//!         .collect(),
-//!     ).unwrap();
+//!    let builder = S3::default()
```

Comment:
it seems that it is an outdated doc test. Change inspired from https://docs.rs/opendal/latest/opendal/services/struct.S3.html#basic-setup